lambda-extensions/config/config.go

```diff
@@ -41,9 +41,13 @@ var validLogTypes = []string{"platform", "function", "extension"}
 
 // GetConfig to get config instance
 func GetConfig() (*LambdaExtensionConfig, error) {
+	sumoHttpEndpoint, ret := os.LookupEnv("SUMO_HTTP_ENDPOINT")
```

Comment:
This is just for logs right? Or do traces also go through this?

Reply:
Yes, just for logs.